assets/js/touchpoints_translations.js

```diff
@@ -0,0 +1,9 @@
+const yesButton = document.querySelector('input[type=submit][value=yes]');
+const noButton = document.querySelector('input[type=submit][value=no]');
+const question = document.querySelectorAll('label')
+const touchpointsRoot = document.getElementById('touchpoints_feedback');
+if (touchpointsRoot) {
+  yesButton.value = touchpointsRoot.dataset.yes;
+  noButton.value = touchpointsRoot.dataset.no;
+  question[1].outerText = touchpointsRoot.dataset.question;
```

Comment:
can we give the label a specific ID or something? Or a class? this is iterating all labels on the page and I worry that adding a form elsewhere on the page could mess this up